docs/source/user-guide/install/index.rst

```diff
@@ -32,9 +33,9 @@ System requirements
 * For Anaconda---Minimum 3 GB disk space to download and install.
 
 
-* Windows, macOS or Linux.
+* Windows, macOS, or Linux.
 
-* Python 2.7, 3.4, 3.5 or 3.6.
```

Comment:
These are only if you install conda from source.  That's really something that only developers working on conda should do.